### PR TITLE
Add a function to get IconCode from IconName

### PR DESCRIPTION
**Pull request by qqli1209**
#### Pull request checklist

- [ ] Include a change request file using `$ npm run change`

#### Description of changes
Adding a function to get IconCode from IconName following this file：
https://github.com/OfficeDev/office-ui-fabric-react/blob/135c06a013468eb717fddab4ccabd6d63274fe4b/packages/styling/src/classNames/IconClassNames.ts

### CODE REVIEW

File: packages/styling/src/styles/IconCodes.ts [hunk 1]
```diff
@@ -2951,3 +2951,11 @@ export const IconCodes = {
   eventDeclined: '\uF425',
 
 };
+
+/**
+ * Get IconCode by IconName.
+*/
+export function getIconCodeByIconName(iconName: string): string {
```
Comment (dzearing) [inline packages/styling/src/styles/IconCodes.ts]: so i have this same thing in my PR, which really changes icons and moves this to a separate icons package. I'd hate for you to take a dependency on this only to have it removed in 5.0. Can we not make this change here?

**Comment by qqli1209** [comment]
@dzearing  Cool!  Could you please tell me where and how to use your new added icons package in your PR?

File: packages/styling/src/styles/IconCodes.ts (hunk 1 above)
Comment (qqli1209) [inline packages/styling/src/styles/IconCodes.ts]: Cool! Could you please tell me where and how to use your new added icons package in your PR?

**Comment by micahgodbolt** [comment]
Closing this up as it appears it'll be coming in via @dzearing's PR.

**Pull request closed without merging**